Extension/package.json

```diff
@@ -3286,6 +3286,17 @@
                         "default": false,
                         "markdownDescription": "%c_cpp.configuration.addNodeAddonIncludePaths.markdownDescription%",
                         "scope": "application"
+                    },
+                    "C_Cpp.onTheFlyDocsEnabled": {
```

Comment:
Nit: I know we've moved away from using "on-the-fly docs" in public communications. The internal setting name does still use that language in VS, but users never really see the raw setting name, unlike VS Code where many users directly edit `settings.json`.

Maybe we should change the name to something like `C_Cpp.copilotHoverEnabled` instead?

Reply:
None of our settings end with "Enabled" and "enabled" is a value of the setting (although we do have .enabled for setting groups). Can we use C_Cpp.copilotHover?

Reply:
updated naming everywhere. Thanks!